tests/test_validate.py

```diff
@@ -350,3 +352,39 @@ def test_validate_dictionnary_no_multiprocess_compatible_warning():
     assert data['metadata']['version'] == u'1.0.0'
     assert data['metadata']['listed'] is True
     assert data['metadata']['id'] == u'my@dict'
+
+def test_validate_hybrid_addon_xpi():
+    """Integration test for the expected notice message on hybrid add-ons."""
+    result = validate(path='tests/resources/validate/hybrid_extension.xpi')
+    data = json.loads(result)
+
+    assert data['success'] is True
+    assert data['errors'] == 0
+    assert data['notices'] == 1
+    assert data['warnings'] == 0
+    assert data['compatibility_summary']
+    assert data['compatibility_summary']['errors'] == 0
+    assert data['compatibility_summary']['notices'] == 0
+    assert data['compatibility_summary']['warnings'] == 0
+    assert data['detected_type'] == 'extension'
+
+    assert len(data['messages']) == 1
+    assert data['messages'][0]['file'] == u'install.rdf'
+    assert data['messages'][0]['message'] == u'This add-on contains an embedded webextension'
+
+    expectedHybridDescriptionRegExp = re.compile (
+        '^The embedded webextension is in the folder "webextension"\. .*'
+        'For more information, refer to https://developer.mozilla.org/.*Embedded_WebExtensions$'
+    )
+    assert expectedHybridDescriptionRegExp.match(data['messages'][0]['description'])
+
+    assert data['message_tree']
+    assert data['signing_summary']['high'] == 0
+    assert data['signing_summary']['medium'] == 0
+    assert data['signing_summary']['low'] == 0
+    assert data['signing_summary']['trivial'] == 0
+    assert data['metadata']
+    assert data['metadata']['name'] == u'name_value'
+    assert data['metadata']['version'] == u'1.2.3.4'
+    assert data['metadata']['listed'] is True
+    assert data['metadata']['id'] == u'an-hybrid-addon@mozilla.com'
```

Comment:
@EnTeQuAk :+1: I was looking for how to do it, that field is not in the properties serialized in the JSON format, I took a deeper look and I found how to do it:
I added an additional step that retrieve the "error bundle" instead of the json result, and I've been able to add an assertion for it :tada: